res/games/gameitem.css

```diff
@@ -0,0 +1,117 @@
+{%- macro playermacro(player,teamalign) %}
+        {% if player.login is not defined %}
+            {% set player_login = player %}
+        {% else %}
+            {% set player_login = player.login %}
+        {% endif %}
+        
+        {% set player_country = "" %}
+        {% set player_global_rating = "???" %}
+        {% if player.countrypath is defined %}
+            {% set player_country = player.countrypath %}
+            {% set player_global_rating = (player.global_rating[0]-(player.global_rating[1]*3))|int %}
+        {% endif %}
+
+        {% set width = 130 %}
+        {% if teams[1] is not defined %}
+            {% set alignment = 'left' %}
+            {% if "-" not in player.login %}
+	            {% set width = 0 %}
+	        {% endif %}
+        {% else %}
+            {% set alignment = 'right' %}
+        {% endif %}
+
+        <tr>
+        {% if teamalign == "left" -%}
+            <td align={{ alignment }} width='{{ width }}'>
+            {{ player_login }}
+            </td>
+            <td>
+            <img src='{{ player_country }}'>
+            </td>
+            <td align={{ alignment }} valign='middle' width='10'>
+            ({{ player_global_rating }})
+            </td>
+        {% else %}
+            <td align='left' valign='middle' width='10'>
```

Comment:
i dont know what you mean again, maybe because the sniplet you commented on is just too small to show what you ment - the "left" here is only for the country icon, it doesnt change a lot between left and right, but it looked better that way when i tested.

and if you mean the width='10': thats only the width that gets assigned to the country icon, i set it to 10 so it doesnt take some standard width but is always as small as possible

Reply:
It's the rating column in both cases, though. I'm just getting the feeling that 'alignment' and 'teamalign' overlap a bit, and that maybe it'd be better to use a variable like 'position' that can be 'first', 'middle' or 'last', and determine alignment and order from it.

Reply:
it seems like most of those positional html formatters were obsolete - now i only have one variable, which is teamaligns, that just decides the order of icon - rating - login name